spec/routing/project_routing_spec.rb

```diff
@@ -60,25 +60,34 @@
   end
 
   describe 'delete' do
-    it { expect(delete('/projects/123')).to route_to(controller: 'projects', action: 'destroy', id: '123') }
+    it { expect(delete('/projects/123')).to     route_to(controller: 'projects', action: 'destroy', id: '123') }
     it { expect(delete('/projects/123.xml')).to route_to(controller: 'projects', action: 'destroy', id: '123', format: 'xml') }
   end
 
-  describe 'miscellanous' do
-    it { expect(get('/projects/123/settings')).to route_to(controller: 'projects',      action: 'settings',     id: '123') }
-    it { expect(get('/projects/123/settings/members')).to route_to(controller: 'projects',      action: 'settings',     id: '123', tab: 'members') }
-    it { expect(put('projects/123/modules')).to route_to(controller: 'projects',      action: 'modules',      id: '123') }
-    it { expect(put('projects/123/archive')).to route_to(controller: 'projects',      action: 'archive',      id: '123') }
-    it { expect(put('projects/123/unarchive')).to route_to(controller: 'projects',      action: 'unarchive',    id: '123') }
+  describe 'miscellaneous' do
+    it { expect(get('/projects/123/settings')).to     route_to(controller: 'projects', action: 'settings', id: '123') }
+    it { expect(put('projects/123/modules')).to       route_to(controller: 'projects', action: 'modules', id: '123') }
+    it { expect(put('projects/123/custom_fields')).to route_to(controller: 'projects', action: 'custom_fields', id: '123') }
+    it { expect(put('projects/123/archive')).to       route_to(controller: 'projects', action: 'archive', id: '123') }
+    it { expect(put('projects/123/unarchive')).to     route_to(controller: 'projects', action: 'unarchive', id: '123') }
```

Comment:
Line is too long. [120/100]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.